apistrap/types.py

```diff
@@ -1,7 +1,21 @@
-from typing import List, Sequence, Iterable
+from dataclasses import dataclass
+from datetime import datetime
+from typing import List, Sequence, Iterable, Union
 
 from schematics.exceptions import BaseError, CompoundError, ConversionError
-from schematics.types import CompoundType
+from schematics.types import CompoundType, StringType, BooleanType
+
+
+@dataclass
```

Comment:
Whoa, bleeding edge!

Reply:
we'll see if @gdynusa and @bedapisl will be able to run python 3.7... if not, we'll wait with the dataclass decorator and implement some constructor

Reply:
I rewrote this so that 3.6 is also supported